# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../koules/mkdirhier.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We do not have this legacy mkdirhier anymore ...

  - Rene Rebe <rene@exactcode.de>

--- ./Imakefile.vanilla	2005-11-04 15:57:54.000000000 +0100
+++ ./Imakefile	2005-11-04 16:00:00.000000000 +0100
@@ -98,16 +98,16 @@
 	@echo "Koules directory is " $(KOULESDIR)
 	@:
 	@echo "Instalin man page to " $(MANDIR)
-	@mkdirhier $(MANDIR)
+	@mkdir -p $(MANDIR)
 	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(MANDIR);
 	@:
 	@echo "Creating directory " $(KOULESDIR)
-	@if [ ! -d $(KOULESDIR) ]; then mkdirhier $(KOULESDIR); fi
+	@if [ ! -d $(KOULESDIR) ]; then mkdir -p $(KOULESDIR); fi
 	@:
 	@if [ "$(INSTALLSOUND)" = "True" ]; 				\
 	 then								\
 		echo "Creating directory " $(SOUNDDIR); 		\
-		if [ ! -d $(SOUNDDIR) ]; then mkdirhier $(SOUNDDIR); fi;    \
+		if [ ! -d $(SOUNDDIR) ]; then mkdir -p $(SOUNDDIR); fi;    \
 		echo "Copying sound server to directory " $(SOUNDDIR); \
 		$(INSTALL) -c -s $(INSTBINFLAGS) $(SOUNDSERVER) $(SOUNDDIR); \
 		echo "Copying sound files into directory " $(SOUNDDIR); \
@@ -126,16 +126,16 @@
 	@echo "Koules directory is " $(KOULESDIR)
 	@:
 	@echo "Instalin man page to " $(MANDIR)
-	@mkdirhier $(MANDIR)
+	@mkdir -p $(MANDIR)
 	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(MANDIR);
 	@:
 	@echo "Creating directory " $(KOULESDIR)
-	@if [ ! -d $(KOULESDIR) ]; then mkdirhier $(KOULESDIR); fi
+	@if [ ! -d $(KOULESDIR) ]; then mkdir -p $(KOULESDIR); fi
 	@:
 	@if [ "$(INSTALLSOUND)" = "True" ]; 				\
 	 then								\
 		echo "Creating directory " $(SOUNDDIR); 		\
-		if [ ! -d $(SOUNDDIR) ]; then mkdirhier $(SOUNDDIR); fi;    \
+		if [ ! -d $(SOUNDDIR) ]; then mkdir -p $(SOUNDDIR); fi;    \
 		echo "Copying sound server to directory " $(SOUNDDIR); \
 		$(INSTALL) -c -s $(INSTBINFLAGS) $(SOUNDSERVER) $(SOUNDDIR); \
 		echo "Copying sound files into directory " $(SOUNDDIR); \
@@ -156,16 +156,16 @@
 	@echo "Koules directory is " $(KOULESDIR)
 	@:
 	@echo "Instalin man page to " $(MANDIR)
-	@mkdirhier $(MANDIR)
+	@mkdir -p $(MANDIR)
 	$(INSTALL) -c $(INSTBINFLAGS) xkoules.6 $(MANDIR);
 	@:
 	@echo "Creating directory " $(KOULESDIR)
-	@if [ ! -d $(KOULESDIR) ]; then mkdirhier $(KOULESDIR); fi
+	@if [ ! -d $(KOULESDIR) ]; then mkdir -p $(KOULESDIR); fi
 	@:
 	@if [ "$(INSTALLSOUND)" = "True" ]; 				\
 	 then								\
 		echo "Creating directory " $(SOUNDDIR); 		\
-		if [ ! -d $(SOUNDDIR) ]; then mkdirhier $(SOUNDDIR); fi;    \
+		if [ ! -d $(SOUNDDIR) ]; then mkdir -p $(SOUNDDIR); fi;    \
 		echo "Copying sound server to directory " $(SOUNDDIR); \
 		$(INSTALL) -c -s $(INSTBINFLAGS) $(SOUNDSERVER) $(SOUNDDIR); \
 		echo "Copying sound files into directory " $(SOUNDDIR); \
